app/models.py

```diff
@@ -50,10 +51,28 @@ class ContactInformation(db.Model):
     postcode = db.Column(db.String, index=False,
                          unique=False, nullable=False)
 
+    def update_from_json(self, data):
+        self.contact_name = data.get("contactName")
+        self.phone_number = data.get("phoneNumber")
+        self.email = data.get("email")
+        self.website = data.get("website")
+        self.address1 = data.get("address1")
+        self.address2 = data.get("address2")
+        self.city = data.get("city")
+        self.country = data.get("country")
+        self.postcode = data.get("postcode")
+
+        return self
+
     def serialize(self):
-        # Should there be links for the associated service(s) / supplier?
+        links = link(
+            "self", url_for(".update_contact_information",
```

Comment:
The link works now, but only for POST. I'm not sure if we need a GET endpoint for individual contact details, but we can always add it later.